### PR TITLE
Control

### PR DESCRIPTION
**Pull request by Maarten-vd-Sande**
**What problem is the PR solving / What's new?**
You can now add a column control and fill it with a name of a fastq, or sample from online. This will then be added as a control :)

**What did change**
One thing to note is that empty values in this column are allowed, which makes for some stupid checks sometimes for nans/strings. But for the rest quite straightforward.

### CODE REVIEW

File: rules/call_peak.smk [hunk 1]
```diff
@@ -33,7 +44,7 @@ rule genrich_pileup:
     computational footprint.
     """
     input:
-        get_genrich_replicates
+        unpack(get_genrich_replicates)
```
Comment (siebrenf) [inline rules/call_peak.smk:47]: in line 64 you call the input `{input.treps}`, so maybe call this `treps = unpack(get_genrich_replicates)` ?

Comment (siebrenf) [reply rules/call_peak.smk:47]: nvm, its a dict. im stupid

File: rules/get_fastq.smk [hunk 2]
```diff
@@ -165,6 +165,7 @@ def get_wrong_fqext(wildcards):
     # exclude samples with the correct fqext
     r1 = wildcards.sample + "_" + config["fqext1"] + "." + config["fqsuffix"] + ".gz"
     r2 = wildcards.sample + "_" + config["fqext2"] + "." + config["fqsuffix"] + ".gz"
+    print(wildcards)
```
Comment (siebrenf) [inline rules/get_fastq.smk:168]: remove

**Comment by Maarten-vd-Sande** [comment]
I started first with control and let @JGASmits test that for me :angel:. And then I started working on the qc here, and decided to already incorporate the control as well. So it got a bit messy.

**Comment by Maarten-vd-Sande** [comment]
Everthing here should be present in the other one as well

**Pull request merged**